LayoutKit/Internal/TextCalculatorHelper.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
```

Comment:
Well wait it looks like you need UIKit in this file. We do successfully link with it under UIKitUtils, so you might want to refactor all UIKit calls to that file instead.